### ISSUE TITLE
Settings - Website accessibility preferences: image labels should be improved.

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#840]
On the contrast adjustments, the screen reader reads "Image" for all the images associated with each option. Better labels could be provided for these images.
